### DISCUSSION TITLE
The DELETE method is not supported for route dashboard/admin/blogs. Supported methods: GET, HEAD, POST.

### DISCUSSION THREAD

**Discussion by devhasanalmamun** [inertiajs/inertia#1836] [Help] [answered]
#### here is my form submission
![image](https://github.com/inertiajs/inertia/assets/69708890/12058928-3bba-444d-ae9c-e312763a19dc)

#### here is route in the admin.php file
![image](https://github.com/inertiajs/inertia/assets/69708890/3099abfc-d804-4f43-a517-1d44af86884a)

#### route list after route:clear
![image](https://github.com/inertiajs/inertia/assets/69708890/2cc7918d-309f-4b12-936e-63f09748bd8f)

#### here is my controller
![image](https://github.com/inertiajs/inertia/assets/69708890/87a758bd-86ee-49e5-93b7-0b8223163c08)

**Comment by RobertBoes** [comment, marked as answer]
You have those routes in `admin.php`, so most likely the routes don't have the Inertia middleware added. Inertia converts a 302 redirect to a 303 so that a follow-up request is made using GET, see https://inertiajs.com/redirects#303-response-code

**Reply by devhasanalmamun** [reply]
@RobertBoes Thanks man you saved me
